server/index.js

```diff
@@ -32,26 +32,49 @@ io.on('connection', function(socket){
 		var lastPlayerAction = lastActions[socket.id];
 		if (lastPlayerAction && new Date().getTime() - lastPlayerAction < 1000){
 			return;
+		}		
+		console.log('can conquer', canConquer(lastPlayerAction, model.map, where, colors[socket.id]));
+		if(canConquer(lastPlayerAction, model.map, where, colors[socket.id])){
+			socket.broadcast.emit('conquered', {
+				id: socket.id,
+				x: where.x,
+				y: where.y,
+				color: colors[socket.id]
+			});
+			socket.emit('conquered', {
+				id: socket.id,
+				x: where.x,
+				y: where.y,
+				color: colors[socket.id]
+			});
+			model.map[where.x][where.y] = colors[socket.id];			
 		}
 		lastActions[socket.id] = new Date().getTime();
 
-		socket.broadcast.emit('conquered', {
-			id: socket.id,
-			x: where.x,
-			y: where.y,
-			color: colors[socket.id]
-		});
-		socket.emit('conquered', {
-			id: socket.id,
-			x: where.x,
-			y: where.y,
-			color: colors[socket.id]
-		});
-		model.map[where.x][where.y] = colors[socket.id];
 	});
 
 });
 
+var canConquer = function (lastAction, map, where, color){
+	if(!lastAction){
```

Comment:
This check is not needed, I think the lastAction param is not needed here either